Godeps/Godeps.json

```diff
@@ -1,11 +1,12 @@
 {
 	"ImportPath": "k8s.io/kubernetes",
-	"GoVersion": "go1.8",
+	"GoVersion": "go1.9",
```

Comment:
Are you running this locally? Master hasn't moved to go1.9 yet.

Reply:
blarch, will revert that part :)